tox.ini

```
@@ -2,7 +2,7 @@
 min_version = 4.0
 requires =
     tox-uv >=0.2.0
-envlist = py3,pypy3,json_infra,static
+envlist = py3,pypy3,json_infra,static,tests_pytest
```

In EEST's `tox.ini`, we used hyphens instead of underscore (e.g. `tests-deployed`, `tests-deployed-benchmark)`: 
```suggestion
envlist = py3,pypy3,json_infra,static,tests-pytest
```

---

I just realized that we now have a discrepancy because "json_infra" is here, which uses underscores, I think we stick to underscores to not break naming in this file.